src/model/preprocess.py

```diff
@@ -47,40 +72,55 @@ def include(content: str) -> bool:
     ]
     for library in libraries:
         a = "import " + library
-        b = "from " + library
+        b = "from " + library + " import"
         if a in content or b in content:
             return True
     return False
 
 
-def get_data():
-    ds = load_dataset(
-        "codeparrot/codeparrot-clean",
-        streaming=True,
-        trust_remote_code=True,
-        split="train",
-    )
-    ds = ds.map(lambda x: {"content": clean_comments(x["content"])})
-    ds = ds.filter(lambda x: include(x["content"]))
-    return ds
-
-
-def keep_only_content(sample: dict) -> dict:
-    # TODO: Is there a way to remove other keys instead of forcing []?
-    for key in sample.keys():
-        if key == "content":
-            continue
-
-        sample[key] = []
-    
-    return sample
-
-
-if __name__ == "__main__":
+# preview cleaning and check
+def preview():
     ds = get_data()
     i = 0
     for x in ds:
         print(x["content"])
         i += 1
         if i == 100:
             break
+
+
+def tests():
```

Comment:
All testing related things should really go in a different file but not the end of the world